tools/releaseBuild/azureDevOps/diagram.puml

```diff
@@ -39,6 +39,8 @@ folder "Package Signing and Upload" as PkgSignUpload {
 folder "Build Test Artifacts" as TestArtifacts {
```

Comment:
Can you add `TestArtifacts` to start and finish?

Reply:
It already is connected to start and finish.